api/model/UserRequest.py

```diff
@@ -0,0 +1,31 @@
+import uuid
+
+class UserRequest:
+	def __init__(self,requestTitle,requestType,requestCategory,requestStatus,username,requestId= str(uuid.uuid4())):
+		self.requestId=requestId
+		self.requestTitle=requestTitle
+		self.requestType=requestType
+		self.requestCategory=requestCategory
+		self.requestStatus=requestStatus
+		self.owner=username
+
+	def getDictionary(self):
```

Comment:
Same here use snake_case for both variables and methods

Reply:
comment seen  thanks for the feedback